scripts/skills/perks/perk_rf_bear_down.nut

```diff
@@ -26,21 +27,36 @@ this.perk_rf_bear_down <- ::inherit("scripts/skills/skill", {
 
 	function onAnySkillUsed( _skill, _targetEntity, _properties )
 	{
-		if (_targetEntity != null && _targetEntity.getSkills().getSkillsByFunction((@(s) this.m.ValidEffects.find(s.getID()) != null).bindenv(this)).len() != 0)
+		if (_targetEntity == null || !this.isSkillValid(_skill))
+			return;
+
+		if (_targetEntity.getSkills().getSkillsByFunction((@(s) this.m.ValidEffects.find(s.getID()) != null).bindenv(this)).len() != 0)
 		{
-			_properties.MeleeSkill += 10;
-			_properties.HitChance[::Const.BodyPart.Head] += 20;
+			_properties.HitChance[::Const.BodyPart.Head] += 33;
+			_properties.ThresholdToInflictInjuryMult *= 0.66;
```

Comment:
This should be a member variable so it can be easier changed/viewed by us or other modders.